.github/workflows/test.yml

```diff
@@ -25,4 +25,6 @@ jobs:
       - name: Run tests with make test
         run: |
           echo "VALEFILES=$(find -L ./docs/ -type d \( -path ./docs/plone.restapi/lib/* -o -path "./docs/plone.restapi/performance/*" \) -prune -false -o -type f -name "*.md" -print)" >> $GITHUB_ENV
+          echo "${{ env.VALEFILES }}"
+          sudo vale sync
           make vale ${{ env.VALEFILES }}
```

Comment:
@stevepiercy maybe:
```
run: |
  VALEFILES=$(find -L ./docs/ -type d \( -path ./docs/plone.restapi/lib/* -o -path "./docs/plone.restapi/performance/*" \) -prune -false -o -type f -name "*.md" -print) make vale
```

(Setting GITHUB_ENV is only needed if you need it to be persisted for subsequent GHA steps. And I suspect using `env` in an expression like that only includes the environment variables that were set when the step started, but I'm not certain.)

Reply:
Fixed in 2202388bae30fdd77846e92a07c2c13809faba35. Thank you!